CONTRIBUTING.md

```diff
@@ -61,7 +61,10 @@ avoid common [go style mistakes](https://github.com/golang/go/wiki/CodeReviewCom
 
 Cluster API uses [GitHub milestones](https://github.com/kubernetes-sigs/cluster-api/milestones) to track releases.
 
-- Minor versions CAN be planned and scheduled twice / calendar year.
+- Major/Minor versions CAN be planned and scheduled twice / calendar year.
```

Comment:
`twice / calendar year.` -> `twice in a calendar year.`

maybe, if MAJOR versions are not planned yet, i'd keep the above sentence about the MINOR cadence and include a note that cadence of MAJOR versions is TBD - e.g. after `1.` is released.

Reply:
+1, let's avoid talking about major releases until we have a need to.